bin/runtime/src/lib.rs

```diff
@@ -272,11 +272,21 @@ parameter_types! {
     pub MaximumMultiplier: Multiplier = Bounded::max_value();
 }
 
+pub struct DivideFeeBy<const N: Balance>;
+
+impl<const N: Balance> WeightToFee for DivideFeeBy<N> {
+    type Balance = Balance;
+
+    fn weight_to_fee(weight: &Weight) -> Self::Balance {
+        Balance::saturated_from(weight.ref_time()).saturating_div(N)
```

Comment:
I feel like we should round `up` here, to never end up with `0` fee. 
So something like 
```
        Balance::saturated_from(weight.ref_time().saturating_add(N-1)).saturating_div(N)
```

Reply:
It should not be possible to end up with 0 _overall_ fee, as fee formula constitute _base_fee_, which is:
* computed [here](https://github.com/Cardinal-Cryptography/polkadot-sdk/blob/aleph-v1.2.0/substrate/frame/transaction-payment/src/lib.rs#L621), 
* initialized [here](https://github.com/Cardinal-Cryptography/polkadot-sdk/blob/aleph-v1.2.0/substrate/frame/system/src/limits.rs#L350),
* finally defined [here](https://github.com/Cardinal-Cryptography/polkadot-sdk/blob/aleph-v1.2.0/substrate/frame/support/src/weights/extrinsic_weights.rs#L57). ie base weight is 124 414 000, which is 0.03% of maximum block weight 400 ms. 
So computed fee should be at least 124 414 000 weight, so in the new code that would be 12 441 400 pico AZERO.

Reply:
Yeah, I thought about `LengthToFee` mostly. With large `N` we would basically make the contribution of length fee `0` if the number of bytes is too small, which is probably no good. But there is already this other problem, that we don't multiply by a constant for length fee.

Reply:
Let's leave is it is, in pallet transaction payment this length fee constituent is only added to total fee, so it should not be a problem